docs/source/checks/tabular/data_integrity/plot_feature_feature_correlation.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
OK just seen that this happens for education and education num. So just need to make sure the condition catches that (define it such that it will), and say something about it